data/data_downloader.py

```diff
@@ -0,0 +1,108 @@
+import os
+
+import pandas as pd
+from astroquery.mast import MastMissions, MastMissionsClass
+
+SECTORS = range(1, 27)
```

Comment:
These are the interesting sectors among all available sectors, I assume? Rename?